### PR TITLE
Feature cli trial validator

### PR DESCRIPTION
**Pull request by rwhite27**

### CODE REVIEW

File: api/routes/api.php [hunk 1]
```diff
@@ -202,4 +202,12 @@
     ]
 ]);
 
+$router->post('/apps-plans/method', [
```
Comment (kaioken) [inline api/routes/api.php]: @rwhite27 rest /apps-plans/{id}/method -> event though we dont use the id

Comment (rwhite27) [reply api/routes/api.php]: ok

File: api/routes/api.php [hunk 2]
```diff
@@ -202,4 +202,12 @@
     ]
 ]);
 
+$router->post('/apps-plans/{id}/method', [
+    'Gewaer\Api\Controllers\AppsPlansController',
+    'updatePaymentMethod',
+    'options' => [
+        'jwt' => false,
```
Comment (kaioken) [inline api/routes/api.php]: @rwhite27 isnt jwt supposed to be true?

File: api/controllers/AppsPlansController.php [hunk 3]
```diff
@@ -258,4 +258,47 @@ public function delete($stripeId): Response
 
         return $this->response($appPlan);
     }
+
+    /**
+     * Update payment method
+     * @return Response
+     */
+    public function updatePaymentMethod(): Response
```
Comment (kaioken) [inline api/controllers/AppsPlansController.php]: @rwhite27 even if we dont use it, pass the id

File: cli/tasks/TrialTask.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+class TrialTask extends PhTask
+{
+    /**
+     * Unset subscription trial_ends_at if trial has ended
+     * @return void
+     */
+    public function unsetTrialEndsAtAction(): void
+    {
+        $subscriptions = Subscription::find([
```
Comment (kaioken) [inline cli/tasks/TrialTask.php]: @rwhite27 dont we need to check if its active ?

File: library/Models/Companies.php [hunk 5]
```diff
@@ -450,6 +450,10 @@ public function startCompanySubscription() : Subscription
             throw new ServerErrorHttpException((string)current($this->getMessages()));
         }
 
+        //Lets create a new default subscription without payment method
+        if (!defined('API_TESTS')) {
```
Comment (kaioken) [inline library/Models/Companies.php]: @rwhite27 remember to check if we can find a way to add hidden .env to scrutinizer

**Pull request merged**